source/Calamari.Tests/Fixtures/ConfigurationTransforms/ConfigurationTransformsFixture.cs

```diff
@@ -27,6 +27,12 @@ public void SetUp()
             configurationTransformer = ConfigurationTransformer.FromVariables(variables, log);
         }
 
+        [Test]
+        public void XmlWithNamespacesIsCorrectlyProcessed()
+        {
+            PerformTest(GetFixtureResource("Samples", "nlog.config"), GetFixtureResource("Samples", "nlog.Release.config"));
```

Comment:
Can we make some assertions about the result of the transform?

Reply:
we could, but before it was blowing up

Reply:
Fixed